app/views/customers/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1> <%= @customer.name %></h1>
+
+<% @customer.items.each do |item| %>
+<%= item.name %>  | 
+$<%= item.price %>  | 
+From <%= item.supermarket.name %> </br>
+<% end %>
+
+</br></br>
+
+<%= form_with url: "/customer_items/", method: :post, data: { turbo: false } do |form| %>
```

Comment:
"/customer_items" is a great RESTful path name, and you could also do "/customers/:customer_id/items" if you wanted to put the customer ID in the path instead of a hidden field.